README.md

```diff
@@ -43,7 +44,7 @@ This provider's versions are able to install and manage the following versions o
 | Kubernetes 1.20 | ✓ | ✓ | ✓ |
 | Kubernetes 1.21 |  | ✓ | ✓ |
 | Kubernetes 1.22 |  |  | ✓ |
- 
+<p align="center"><img alt="capi" src="https://cloud.google.com/_static/cloud/images/favicons/onecloud/super_cloud.png" width="192x" /></p> 
```

Comment:
ok, this is not clear, why you are moving the image to below? they should be side-by-side

and looking at the current page there is no issue

I will close this PR, because this is not fixing or changing anything.
thanks again for the work on this